pkgs/applications/misc/latte-dock/default.nix

```diff
@@ -10,13 +10,15 @@ mkDerivation {
     owner = "psifidotos";
     repo = "Latte-Dock";
     rev = "v${version}";
-    sha256 = "1967hx4lavy96vvik8d5m2c6ycd2mlf9cmhrv40zr0784ni0ikyv";
+    sha256 = "0vdmsjj1qqlzz26mznb56znv5x7akbvw65ybbzakclp4q1xrsrm2";
   };
 
-  buildInputs = [ plasma-framework xorg.libpthreadstubs xorg.libXdmcp ];
+  buildInputs = [ plasma-framework xorg.libpthreadstubs xorg.libXdmcp xorg.libSM ];
+
+  #  cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Debug" ];
```

Comment:
Is this still required?

Reply:
If you mean the cmakeFlags comment, then no. I had some issues having latte run and it's an unintended leftover from testing.

Reply:
Also, I'm probably messing something up with the differences between buildInputs and nativeBuildInputs :(